packages/react-components/src/components/ParticipantList.tsx

```diff
@@ -321,6 +321,10 @@ export const ParticipantList = (props: ParticipantListProps): JSX.Element => {
           itemProps: {
             styles: props.styles?.participantItemStyles?.participantSubMenuItemsStyles
           },
+          iconProps: {
+            iconName: 'ContextMenuRemoveParticipant',
+            styles: { root: { lineHeight: 0 } }
```

Comment:
nit : I think there is a style that we have set you can import so there isn't a magic 0